drivers/sdp3x/include/sdp3x_params.h

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2019 Dirk Ehmen
+ *               2020 Nishchay Agrawal
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_sensors
+ * @{
+ * @file
+ * @brief       Device driver params interface for the SDP3x sensor.
+ *
+ * @author      Dirk Ehmen       <ehmen@ibr.cs.tu-bs.de>
+ * @author      Nishchay Agrawal <f2016088@pilani.bits-pilani.ac.in>
+ * @}
+ */
+
+#ifndef SDP3X_PARAMS_H
+#define SDP3X_PARAMS_H
+
+#include "board.h"
+#include "sdp3x.h"
+#include "saul_reg.h"
```

Comment:
```suggestion
#include "saul_reg.h"
#include "periph/gpio.h"
```

Reply:
I don't think this is required, as it works without include "periph/gpio.h"

Reply:
Doesn't mean it works in the future ;) If you are using gpio_* functions you should have the appropriate include.

Reply:
Oh yeah, ok. :)